addons/hr_holidays/models/hr_leave.py

```diff
@@ -1023,9 +1023,9 @@ def _prepare_holidays_meeting_values(self):
                 'res_id': holiday.id,
             }
             # Add the partner_id (if exist) as an attendee
-            if user and user.partner_id:
+            if holiday.employee_id and holiday.employee_id.work_contact_id:
                 meeting_values['partner_ids'] = [
-                    (4, user.partner_id.id)]
+                    (4, holiday.employee_id.work_contact_id.id)]
             result[user.id].append(meeting_values)
```

Comment:
As discussed, a nice fallback system where the current behavior is used when the user is available would be better to avoid breaking existing flows. :-)